lib/provider.js

```diff
@@ -1,137 +1,158 @@
-/*
- * decaffeinate suggestions:
- * DS101: Remove unnecessary use of Array.from
- * DS102: Remove unnecessary code created because of implicit returns
- * DS103: Rewrite code to no longer use __guard__, or convert again using --optional-chaining
- * DS104: Avoid inline assignments
- * DS207: Consider shorter variations of null checks
- * Full docs: https://github.com/decaffeinate/decaffeinate/blob/main/docs/suggestions.md
- */
+
 const fs = require('fs');
 const path = require('path');
 
 const CLASSES = require('../completions.json');
 
 const propertyPrefixPattern = /(?:^|\[|\(|,|=|:|\s)\s*(atom\.(?:[a-zA-Z]+\.?){0,2})$/;
 
-module.exports = {
-  selector: '.source.coffee, .source.js',
-  filterSuggestions: true,
+let packageDirectories = [];
```

Comment:
Moving to this style prevents overwriting `packageDirectories` directly and therefor would be a lost of hackability. Same thing all other variables and functions that are exported by the module.